### PR TITLE
Update minSdkLevel to 15. Remove all obsolete code and resources. #392

### PR DESCRIPTION
**Pull request by lemonboston**
A lot of code and resources could be removed.

### CODE REVIEW

File: opentasks/src/main/java/org/dmfs/tasks/utils/BaseActivity.java [hunk 1]
```diff
@@ -83,24 +81,15 @@ protected void onSaveInstanceState(Bundle outState)
...
     {
         super.onPause();
-        /*
-         * On older SDK version we have to store permanent data in onPause(), because there is no guarantee that onStop() will be called.
-		 */
-        if (VERSION.SDK_INT < VERSION_CODES.HONEYCOMB)
-        {
-            RetentionMagic.persist(this, mPrefs);
-        }
+        RetentionMagic.persist(this, mPrefs);
```
Comment (dmfs) [inline opentasks/src/main/java/org/dmfs/tasks/utils/BaseActivity.java]: This should be removed. It was only enabled on SDK<11.

**Comment by dmfs** [comment]
@lemonboston please rebase

File: opentasks/src/main/java/org/dmfs/tasks/utils/BaseActivity.java (hunk 1 above)
Comment (lemonboston) [reply opentasks/src/main/java/org/dmfs/tasks/utils/BaseActivity.java]: Good catch, thanks, I've corrected it.

Squashed and rebased, ready for check.

**Comment by lemonboston** [comment]
Yep, this is the best :)

**Pull request merged**